src/main/scala/com/microsoft/sqlserver/jdbc/spark/connectors/ReliableSingleInstanceStrategy.scala

```diff
@@ -36,7 +36,7 @@ object ReliableSingleInstanceStrategy extends  DataIOStrategy with Logging {
    * Phase 2 Driver combines all staging tables to transactionally write to user specified table.
```

Comment:
Add test results please post test addition. 2 things to look at 1. All old test are running fine.  2. new test work fine.

Reply:
Added.